common/mock-local-home-platform.ts

```diff
@@ -0,0 +1,127 @@
+/*
+ * Mock Local Home Platform class
+ * - Interacts with bundled HomeApp
+ * - Accepts scan config
+ * - Interacts with virtual network
+ */
+
+/// <reference types="@google/local-home-sdk" />
+import {
+  MockNetwork,
+  MockUDPListener,
+  DiscoveryData,
+  RemoteAddressInfo,
+} from './mock-radio';
+import cbor from 'cbor';
```

Comment:
This shouldn't be needed by the platform mock as this is just a protocol that happen to be used by the sample

Reply:
Will remove

Reply:
Done